lib/fizzbuzz.rb

```diff
@@ -1,9 +1,10 @@
 class SuperFizzBuzz
 
   def run(input)
-
-    #Implement your code here
-
+    result = ""
+    result << "Fizz" if input % 3 == 0
+    result << "Buzz" if input % 5 == 0
+    result
   end
```

Comment:
This is an elegant solution @jaybobo! I really like that this reduces the explicit variable declaration for something that is ultimately just going to get returned. Nicely done!